analytics.py

```diff
@@ -0,0 +1,284 @@
+import math     #For all of the math used
+from .. import utils
+#------------------------------------------------------------------
+#
+# Joseph Cruz
+# analytics.py
+#
+# Stores the functions from point_pattern.py that perform analysis
+# on data.
+#------------------------------------------------------------------
+
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+#
+# function mean_nearest_neighbors_randomPoints
+#
+# Determines the mean nearest neighbor of randomly generated points
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+
+def mean_nearest_neighbors_randomPoints():
```

Comment:
Here you can use a function signature like: `mean_nearest_neighbors_randomPoints(p=99, n=100)`.  This way you can avoid needing to have input via a command line - someone can alter parameters programmatically.